atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -1218,7 +1220,7 @@ private void createAndInitializeSweepersAndWaitForOneBackgroundIteration(int swe
     }
 
     private void waitUntilSweepRunsOneIteration() throws InterruptedException {
-        Thread.sleep(3000L);
+        Thread.sleep(4000L);
```

Comment:
"fine tuning"

Reply:
Yeah, the "regression" this PR introduces is a bit more than I expected, and it's not safe to give numbers greater than 5000 or else in-memory KVS will fail because there's enough time to run more than one iteration.